### DISCUSSION TITLE
Cannot find ".xcodeproj/../YES"

### DISCUSSION THREAD

**Discussion by maksimvm** [fastlane/fastlane#20062] [Q&A]
I got an error while running fastlane increment_build_number, I tried to specify path to .xcodeproj, but still got this type of error. Can anyone explain what parameters I should fix, provide to fix this issue?
![Screenshot 2022-03-10 at 12 22 05](https://user-images.githubusercontent.com/25157397/158299825-7a5c7c07-040b-4844-adf3-abc625801de6.png)

**Comment by kyle-bai** [comment]
I ran into this exact issue after adding unit tests to my project. Have you added anything recently? I resolved it by running "bundle update fastlane" then manually updating the "CURRENT_PROJECT_VERSION" variable for the unit tests in the project.pbxproj file to match the version of the existing targets.

Personally, I don't like this answer because it doesn't explain what the problem was but it was resolved after completing the steps listed above.

**Reply by maksimvm** [reply]
Hello. Thank you for response, yes, I have added UI Tests to my project and removed all cocoapods dependencies (moved to SPM) , and after that I started receiving this warning.
I tried "bundle update fastlane", and then I checked "CURRENT_PROJECT_VERSION" variable in the project.pbxproj file (the version number was the same everywhere), but I'm still getting this warning. I think that in my case it happens because of cocoapods removal, but I still have no idea how to fix this issue.

**Comment by bigmouthstrks** [comment]
Any updates about this issue? I'm having the same problem when attempting to tun increment_version_number(version_number: "x.x.x").
I tried using the agvtool command directly and seems like the "Cannot find ABC.xcodeproj/../YES" output comes from agvotool and not from Fastlane itself.

**Comment by rkrim** [comment]
Hi, 
This is related to a property about **Info.plist**.
It's neither generated by **increment_build_number** action, nor by **fastlane**.

It's the output result of "**agvtool**", the Apple-generic versioning tool for Xcode projects.
The tool used by **increment_build_number** action.

You can get rid of this with or without XCode:

With XCode:
 - Open Xcode
 - In the Navigator, click on the project
 - Select your target
 - Select the "Build Settings" tab
 - Search for "GENERATE_INFOPLIST_FILE"
 - Change the value to "No". Or hit backspace on the keyboard to reset to default value "No" .
 - You need to do this for each target

Without XCode:
- Open the .xcodeproj > project.pbxproj with any text editor
- Search for `GENERATE_INFOPLIST_FILE`.
- Either remove the line or set the value to NO.
  `GENERATE_INFOPLIST_FILE = NO;`
- You can find multiple lines, one per target. You need to do this for all

**Reply by goodyboy6** [reply]
i have the same problem, and not resolve yet.  i set  all to "GENERATE_INFOPLIST_FILE = NO;"  then tell me : 
Cannot find "xxx.xcodeproj/../NO"

anyone resolve?

**Reply by paresh-thakor** [reply]
Same issue for me as well. Don't know how to solve these things.

**Reply by ashitanojoe** [reply]
I was able to get past this error by removing the `GENERATE_INFOPLIST_FILE` line from the `project.pbxproj`. Changing the value to `NO` gave the same result as @goodyboy6.

**Reply by Minny27** [reply]
I got the same issue. No matter how value of `GENERATE_INFOPLIST_FILE` assigned `YES` or `NO`, It is not solved.
It only repeated `Cannot find ...`

**Comment by ashitanojoe** [comment]
Have the same issue after adding a Notification Service extension. Been trying to find a workaround all day. No luck.

**Reply by mylogon341** [reply]
did you find out how to fix this? I'm also having this issue after creating a notification service extension and it's just having a bad time.

**Reply by ashitanojoe** [reply]
Removing the `GENERATE_INFOPLIST_FILE` lines from `project.pbxproj` file manually with a text editor resolved this issue for me.

**Reply by nilekell** [reply]
2 years later... this does not work for me :/

**Reply by standinga** [reply]
I have the same issue with Notification Service extension. If I set the `GENERATE_INFOPLIST_FILE` to NO, I'm getting: `Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.`
